prettify-ts-plugin/src/main.ts

```diff
@@ -0,0 +1,8 @@
+import type * as ts from 'typescript'
+import type * as tsserver from 'typescript/lib/tsserverlibrary'
+
+import { PluginCreator } from './pluginCreator'
+
+export = function init (modules: { typescript: typeof ts }): tsserver.server.PluginModule {
```

Comment:
I would fold in `pluginCreator.ts` into this file if it weren't for the fact that `export =` seems required. It doesn't pick up on `export default` or alternatives and I wanted to make the types shareable.